src/mame/nintendo/vsnes.cpp

```diff
@@ -1221,7 +1221,7 @@ INPUT_PORTS_END
 static INPUT_PORTS_START( vsnes_zapper )
 	PORT_START("IN0")
 	PORT_BIT( 0x0f, IP_ACTIVE_HIGH, IPT_UNUSED )
-	PORT_BIT( 0x10, IP_ACTIVE_LOW,  IPT_UNUSED )            // low 6 bits always read 0b010000
+	PORT_BIT( 0x10, IP_ACTIVE_LOW,  IPT_UNUSED )            // ALM wire. 1: gun connected, 0: disconnected alarm (Duck Hunt and Hogan's Alley only?)
```

Comment:
Should this be an machine configuration input so it can be exercised, since it isn’t actually unused?

Reply:
Sure. It's probably better documentation as a live config. Until recently there was `m_sound_fix` counter that inserted the alarm's 1 bit into the appropriate shift register at the appropriate time. I wonder if anybody realized that horn sound in Duck Hunt/Hogan's Alley was supposed to be an alarm?